cmd/app/internal/service/tts-stt/tts-stt.py

```
@@ -31,9 +35,59 @@
+# except Exception as e:
+#     logger.warning(f"GPU failed: {e}. Falling back to CPU.")
+#     model = WhisperModel("base", device="cpu", compute_type="int8")
+
+# Initialize  TTS models
+logger.info("Loading TTS models...")
+
+try:
+    tts_fast = TTS("tts_models/en/ljspeech/vits")
+    tts_fast.to("cuda")
+    logger.info("VITS model loaded successfully")
+except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)